### PR TITLE
Rhmap 7754

### PR DESCRIPTION
**Pull request by pb82**
# Motivation

The `appdata export status` command should not poll jobs that have failed. Exit with an appropriate error message.

ping @trepel @ziccardi @wei-lee

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by jcesarmobile** [comment]
LGTM

**Comment by trepel** [comment]
Yes, as far as I can tell, it looks good.

File: lib/cmd/fh3/appdata/export/status.js [hunk 1]
```diff
@@ -97,6 +98,12 @@ module.exports = {
 
         // supercore sends the actual result in the `data` field
         job = job.data;
+
+        // Don't wait for failed jobs
+        if (job.status === STATUS_FAILED) {
+          return cleanupAndExit(new Error("Cannot query progress: Job has failed"));
```
Comment (ziccardi) [inline lib/cmd/fh3/appdata/export/status.js]: Messages should be passed to i18n._() to allow internalization

**Comment by ziccardi** [comment]
LGTM. Just added one comment

**Pull request merged**